### ISSUE TITLE
Vortice Mathematics 3.1.18 has conflict issues with V1.9.41 Vortice Windows

### ISSUE THREAD

**Issue by h1cks** [amerkoleci/Vortice.Windows#133]
Installing the latest version of Vortice mathematics give me a number of errors relating to linking.

If I revert to 3.1.14 then the issue disappears.  

Below, sample of errors

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0012	The type 'Color4' is defined in an assembly that is not referenced. You must add a reference to assembly 'Vortice.Mathematics, Version=1.3.0.0, Culture=neutral, PublicKeyToken=null'.	Empire Redux (net5.0-windows)	D:\Development\Empire Redux\Empire Redux\GPUCode\Shader Classes\HUDShader.cs	143	Active
```

**Comment by andrew-boyarshin** [comment]
3.1.18 is strong-name signed, 1.9.41 is not. This causes assembly binding to fail. @amerkoleci maybe release a version (at least preview one) that is based on latest main branch (with strong name signing)?

**Comment by amerkoleci** [comment]
Give me 2 hours and I will release new nuget version

**Comment by amerkoleci** [comment]
> 3.1.18 is strong-name signed, 1.9.41 is not. This causes assembly binding to fail. @amerkoleci maybe release a version (at least preview one) that is based on latest main branch (with strong name signing)?

https://github.com/amerkoleci/Vortice.Mathematics/runs/2316304495?check_suite_focus=true

What happened?

**Comment by andrew-boyarshin** [comment]
Feels like NuGet feed is gone.

**Comment by andrew-boyarshin** [comment]
Try adding <https://github.com/dotnet/templating/blob/main/template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Nuget/nuget.config> to the repo root? My working theory is that hosted GitHub Actions agent pool images were updated and now have NuGet.config misconfiguration.

**Comment by andrew-boyarshin** [comment]
Confirmed. <https://github.com/actions/virtual-environments/issues/3038>

**Comment by amerkoleci** [comment]
Now it works.

**Comment by andrew-boyarshin** [comment]
Btw, I meant releasing a new Vortice.Windows version, not the Vortice.Mathematics. To make both packages strong name signed.

**Comment by amerkoleci** [comment]
Just released 1.9.45 stable release on nuget, please try it.
